File: Programación/Tema 10 - Control y Manejo de Excepciones/AppInstituto2/src/Alumno.java
public class Alumno extends Miembro {

    private String cicloActual;
    private double mediaNotas;
    private double importeMatricula;


    public Alumno() {
        this.cicloActual = "";
        this.mediaNotas = 8.5;
        this.importeMatricula = 20.5;
    }

    public Alumno(String dni, String nombre, int edad, String direccion, String cicloActual, double mediaNotas, double importeMatricula) {
        super(dni, nombre, edad, direccion);
        this.cicloActual = cicloActual;
        this.mediaNotas = mediaNotas;
        this.importeMatricula = importeMatricula;
    }
}

File: Primer año/Contornos Cliente/Tema 3. JavaScript Avanzado/Practicas/Practica 2/js/Alumno.js
class Alumno {
    constructor(nombre, direccion, dni) {
        this.nombre = nombre;
        this.direccion = direccion;
        this.dni = dni;
    }




    //Getters y setters
    getNombre() { return this.nombre; }
    setNombre(nombre) { this.nombre = nombre; }

    getDireccion() { return this.direccion; }
    setDireccion(direccion) { this.direccion = direccion; }

    getDNI() { return this.dni; }
    setDNI(dni) { this.dni = dni; }

}

